demo

```diff
@@ -14,17 +14,22 @@ docker-compose, if you want to run the demo, run:
 EOF
 }
 
-CLIENT="-f docker-compose.client.yml"
+CLIENT=
+CLIENT_ARGS="-f docker-compose.client.yml"
 while [ -n "$1" ]; do
-    if [ -n "${NO_CLIENT}" -o "$1" = "--no-client" ]; then
-        unset CLIENT
-        echo "-- disabling client containers"
-        if [ -n "$NO_CLIENT" ]; then
-            unset NO_CLIENT
-            continue
-        fi
+    if [ "$1" = "--no-client" ]; then
+        echo "--no-client argument is deprecated. Client is now disabled by default and can be enabled with --client"
```

Comment:
should this also be reflected in the `usage` description? I saw that the `--client` arg would be capable of defining the number of devices, but don't seem to understand how this would work - maybe put that instead?

Reply:
Oops, you're right. Will fix that.